scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script was written as part of a project to migrate
+# split repositories to a mono repository google-cloud-python.
+# This script will delete all files in a split repository and
+# update the README to indicate that the repository is archived.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>
+
+Args:
+   split-repo-name: The name of the split repository. For example, \`python-access-approval\`.
+   artifact-name: The name of the artifact on PyPI. For example, \`google-cloud-access-approval\`.
+"
+  exit 1
```

Comment:
If the user sources this script, this will exit this running script. So either enforce at the top that the user is executing rather than sourcing or define `$EXIT` appropriately and use it in place of `exit`:

```
[[ "${BASH_SOURCE[0]}" != "${0}" ]] && EXIT=return || EXIT=exit
```

More info:  https://stackoverflow.com/a/28776166

Reply:
Added commit 3e39ef4 which is similar to the changes in https://github.com/googleapis/google-cloud-python/pull/11106/files